.changeset/long-icons-pull.md

```diff
@@ -0,0 +1,37 @@
+---
+'@xstate/react': patch
+---
+
+Correct reference to the machine in `useInterpret` hook.
```

Comment:
This is an implementation detail - it doesn't quite matter for our consumers so it's best to leave this out
```suggestion
```

Reply:
How do we word it so that the consumers are aware of a bug fix?

Reply:
The other suggestion I've made mentions that it's a fix for a regression